docs/crukmi.md

```diff
@@ -0,0 +1,15 @@
+# nf-core/configs: Cancer Research UK Manchester Institute Configuration
+
+All nf-core pipelines have been successfully configured for the use on the HPC (phoenix) at Cancer Research UK Manchester Institute.
+
+To use, run the pipeline with `-profile crukmi`. This will download and launch the [`crukmi.config`](../conf/crukmi.config) which has been pre-configured with a setup suitable for the phoenix HPC. Using this profile, singularity images will be downloaded to run on the cluster.
```

Comment:
Phoenix?

Reply:
The HPC at CRUK-MI is called "phoenix" (we had a major fire in 2017...)